### PR TITLE
Removing office_initiatives styles

### LINKED ISSUE #783: Are `office_initiatives` styles still in use?
Trying to find where https://github.com/cfpb/cfgov-refresh/blob/flapjack/src/static/css/pages/offices.less#L14-L36 are used and failing. Are they still in use?

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Closes #783.

## Removals

- Removed extraneous css not in use by project 
## Testing

- Visit any office page with an "Our Work" section

## Review

- @sebworks 
- @anselmbradford 
- @jimmynotjim

### CODE REVIEW

**Comment by sebworks** [comment]
can we git rid of `. initiative` as well?

**Comment by KimberlyMunoz** [comment]
It looks like it's still being used by the FOIA Records page, but I bet that can be easily refactored. Let me know if you want me to handle that in this PR or the next.

**Comment by sebworks** [comment]
no, let's keep this one pure. :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Removed requestAnimationFrame polyfill.
 - Removed `_tests/browser_tests/README.md`, `_tests/macro_testing/README.md`, `_tests/processor_tests/README.md`.
 - Removed `grunt vendor` from `setup.sh`.
+- Removed unused CSS
```
Comment (anselmbradford) [inline CHANGELOG.md]: No worries with updating this, but a quick note on the area where the CSS was removed may make the changelog more useful as a reference down the road. (see last entry in 0.2.0. https://github.com/cfpb/cfgov-refresh/blob/flapjack/CHANGELOG.md#fixed-10 :stuck_out_tongue_closed_eyes: )

Comment (KimberlyMunoz) [inline CHANGELOG.md]: How about I fix this one in the next PR where I address Steven's feedback to keep it clean? :)

Comment (KimberlyMunoz) [inline CHANGELOG.md]: Fixed in #800.

**Pull request merged**